app/utils/sentry.ts

```diff
@@ -39,9 +40,18 @@ export function initializeSentry() {
         return;
     }
 
+    const mmConfig = {
+        environment: isBetaApp ? 'beta' : 'production',
+        tracesSampleRate: isBetaApp ? 1.0 : 0.2,
+        sampleRate: isBetaApp ? 1.0 : 0.2,
+        attachStacktrace: isBetaApp, // For Beta, stack traces are automatically attached to all messages logged
```

Comment:
@enahum  I ended removing the previous `loglevel` that was here.  That property didn't exist and for the fatal errors, I'm checking them like that https://github.com/mattermost/mattermost-mobile/blob/d10d8cda7c6a22d4ab849836b4b33cabbd5d4782/app/utils/error_handling.ts#L47

Reply:
Right, but here we should also add it